examples/with_winit/src/lib.rs

```diff
@@ -444,6 +464,8 @@ fn run(
                             width,
                             height,
                             antialiasing_method,
+                            #[cfg(feature = "debug_layers")]
```

Comment:
We should probably warn if the debug_layers feature is enabled, but `args.async_pipeline` isn't set.